app/styles/shared/gradient.css

```diff
@@ -0,0 +1,23 @@
+.bg-gradient__container {
```

Comment:
Instead of using custom class with BEM and not following Tailwind convention

Better to just put all of these classes into the component as it's only being used once in a component